ui/packages/catalog-ui-search/src/main/webapp/js/ApplicationSetup.js

```diff
@@ -83,10 +83,8 @@ let getErrorResponse = function(event, jqxhr, settings, throwError) {
     settings.type === 'GET'
   ) {
     return {
-      title: `Error Polling ${properties.i18n['Sources']}`,
-      message: `Unable to query server for list of active ${
-        properties.i18n['sources']
-      }`,
+      title: properties.i18n['sources.polling.error.title'],
+      message: properties.i18n['sources.polling.error.message']
```

Comment:
Is there any more context we can give to the user about why we could not query the server for the list of active sources?

Reply:
Unfortunately I don't think so. The server would have to send back more info about the error to do this.